ci-operator/step-registry/openstack/provision/bastionproxy/openstack-provision-bastionproxy-commands.sh

```diff
@@ -80,14 +94,15 @@ openstack security group rule create --ingress --protocol tcp --dst-port 22 --de
 openstack security group rule create --ingress --protocol tcp --dst-port 8213 --description "${CLUSTER_NAME} squid" "$sg_id" >/dev/null
 >&2 echo "Security group rules created in ${sg_id} to allow SSH and squid access"
 
-server_id="$(openstack server create -f value -c id $ZONES_ARGS \
+openstack server create --wait $ZONES_ARGS \
 		--image "$BASTION_IMAGE" \
 		--flavor "$BASTION_FLAVOR" \
-		--network "$NET_ID" \
+		--network "$NET_ID" $PROV_NET_ARGS \
```

Comment:
If the bastion also uses the provider Network maybe the tenant network w/fip is not needed?

Reply:
it is needed because the provider network is not reachable from outside (prow), so you need a tenant network for bastion, with a router, and a FIP assigned to the bastion server, so prow can reach the bastion proxy and therefore reach the provider network.

Reply:
got it and that is why we use sshuttle